### PR TITLE
FreeBSD build fixes

### PR DESCRIPTION
**Pull request by jhixson74**
commit a80eca5af8142e39e938bf8010000550bdd7884c broke mc from building
on FreeBSD. This commit seperates out the files to have individual NetBSD files.

### CODE REVIEW

File: cmd/client-fs_freebsd.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// +build freebsd netbsd
+// +build freebsd
```
Comment (klauspost) [inline cmd/client-fs_freebsd.go:1]: This can be omitted now, the filename is guarding it now.

File: cmd/client-fs_netbsd.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
+// +build netbsd
```
Comment (klauspost) [inline cmd/client-fs_netbsd.go:1]: Not needed

File: cmd/client-fs_netbsd.go [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+/*
+ * MinIO Client (C) 2017 MinIO, Inc.
```
Comment (klauspost) [inline cmd/client-fs_netbsd.go:4]:
```suggestion
 * MinIO Client (C) 2020 MinIO, Inc.
```

File: pkg/disk/stat_freebsd.go [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// +build freebsd netbsd
+// +build freebsd
```
Comment (klauspost) [inline pkg/disk/stat_freebsd.go:1]: Can be omitted.

File: pkg/disk/stat_netbsd.go [hunk 5]
```diff
@@ -0,0 +1,63 @@
+// +build netbsd
```
Comment (klauspost) [inline pkg/disk/stat_netbsd.go:1]: omit

**Review by klauspost — COMMENTED** [review]
Thanks for your contribution!

I just have a few cosmetic changes. The rest LGTM.

**Comment by jhixson74** [comment]
Thanks for the review. I've updated the PR with the suggested fixes!

**Review by harshavardhana — APPROVED** [review]

**Review by klauspost — APPROVED** [review]
LGTM

**Pull request merged**